chain/chain/src/chain.rs

```diff
@@ -1799,32 +1800,38 @@ impl Chain {
         Ok(())
     }
 
-    pub fn set_state_finalize(
+    pub fn start_set_state_finalize(
```

Comment:
maybe rename this function to `schedule_apply_state_parts`, I think that describes better what this function does